Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "I_BatteryStateOfChargeService.h"
+#include "QTime"
```

Comment:
Including third party libraries should be done with angled brackets - `#include <QTime>`